### DISCUSSION TITLE
How to become a Djangonaut?

### DISCUSSION THREAD

**Discussion by priyapahwa** [djangonaut-space/program#15] [Q&A] [answered]
Hey there, I am super amazed by the program thus I'd like to know the details of how and when to join the upcoming cohort.

**Comment by tim-schilling** [comment]
Hi @priyapahwa, thanks for asking. I'm thrilled that you're interested in joining. We're still figuring out the logistics of the program, including the application process. We will share those details when they are available, but it may be some time. Our pilot program is slated to end in October, so the next cohort would be after that at least.

**Comment by priyapahwa** [comment]
Alright, thanks for the heads-up @tim-schilling. Eagerly looking forward to it 🥳 
P.S. Let me know if I can contribute anywhere in building up the program, I'd be more than happy.

**Comment by RachellCalhoun** [comment, marked as answer]
Hi @priyapahwa,  I wanted to follow up with you as we are planning out the future sessions! Although we are still finalizing the details for the next program session, like the dates and application process, but if you'd like to get an email when it's ready to go please fill out this google form: https://forms.gle/4vNuqQwEK6o48WqHA and we'll reach out again before we start our next session. Hope to hear from you soon :)

**Reply by priyapahwa** [reply]
Hey there,
Thanks for getting back. Looking forward to it.
